packages/aws-utils/mapping-templates/scan-request.txt

```diff
@@ -0,0 +1,27 @@
+#set($expression = "")
+#set($expressionValues = {})
+#set($expressionNames = {})
+#foreach( $filter in $context.arguments.query )
+  #set( $expression = "${expression} $filter.get('expression')" )
+  #set( $values = $expressionNames.put("#$filter.get('expressionName')", $filter.get('expressionName')) )
+  #if($filter.get("expressionStringValue"))
+    #set( $value =  $filter.get("expressionStringValue") ) 
+  #else
+    #set( $value =  $filter.get("expressionNumberValue") ) 
+  #end
+  #set( $values = $expressionValues.put(":$filter.get('expressionName')", $util.dynamodb.toDynamoDB($value)) )
+  #if ( $foreach.hasNext )
+  #set( $expression = "${expression} AND" )
+  #end
+#end
+{
+  "version" : "2017-02-28",
+  "operation" : "Scan",
+  "filter" : {
+    "expression": "${expression}",
+    "expressionNames" : $utils.toJson($expressionNames),
```

Comment:
Random note ... Looks like you use $utils and $util interchangeably but the docs only talk about `$util` . Looks like a documentation issue or some kind of backwards compat support.